Content.Shared/Audio/AmbientSoundComponent.cs

```diff
@@ -47,5 +47,6 @@ public sealed class AmbientSoundComponentState : ComponentState
         public bool Enabled { get; init; }
         public float Range { get; init; }
         public float Volume { get; init; }
+        public SoundSpecifier? Sound { get; init; }
```

Comment:
Have you checked that the system will not freak out if this is null?

Reply:
oh nvm this is just on the state update, that's fine

Reply:
An alternative that also works:
`public SoundSpecifier Sound { get; init; } = null!;`
Ideally, it should not be nullable, but I'm not sure if this is any better?

Reply:
it's fine, I did some debugging myself